themes/openy_themes/openy_carnation/src/js/openy_carnation.js

```diff
@@ -0,0 +1,460 @@
+/**
+ * @file
+ * OpenY Carnation JS.
+ */
+(function ($) {
+
+  "use strict";
+  Drupal.openyCarnation = Drupal.openyCarnation || {};
+
+  /**
+   * Alert Modals
+   */
+  Drupal.behaviors.openyAlertModals = {
+    attach: function (context, settings) {
+      var alertModals = $('.alert-modal', context);
+
+      $(window).on('load',function() {
+        if (alertModals.length) {
+          alertModals.modal('show');
+        }
+      });
+    }
+  };
+
+  /**
+   * Move Header Banner paragraph to header.
+   */
+  Drupal.behaviors.openyBanners = {
+    attach: function (context, settings) {
+      var bannerHeader = $('.paragraph--type--banner, .landing-header, .page-heading');
+      if (bannerHeader.length > 0) {
+        $('.banner-zone-node').once('openy-banners').append(bannerHeader.eq(0));
+        $('body').addClass('with-banner');
+      } else {
+        $('body').addClass('without-banner');
+      }
+    }
+  };
+
+  /**
+   * Ensure breadcrumbs are after banners in the DOM
+   */
+  Drupal.behaviors.moveBreadcrumbs = {
+    attach: function (context, settings) {
+      var breadCrumbs = $('.breadcrumbs-wrapper', context);
+      var bannerCta = $(
+        '.banner-zone-node .banner .banner-cta, ' +
+        '.banner-zone-node .landing-header, ' +
+        '.banner-zone-node .paragraph-gallery', context);
+
+      if (breadCrumbs.length && bannerCta.length) {
+        breadCrumbs.once('openy-breadcrumbs').appendTo(bannerCta);
```

Comment:
Here we move HTML around again. Lets discuss this too.

Reply:
Breadcrumbs are outputted funny in the carnation theme. We basically place it below the banners if they exists, but again they need to be appended to the main "banner-zone" div. I'm not sure how else we can accomplish this without moving it via jQuery.